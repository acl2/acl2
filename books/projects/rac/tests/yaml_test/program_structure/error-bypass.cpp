#include <array>
using namespace std;

// RAC begin

int foo(int a, array<int, 3> b) { return a && b; }
int bar(int a, array<int, 3> b) { return a && b; }

// RAC end
