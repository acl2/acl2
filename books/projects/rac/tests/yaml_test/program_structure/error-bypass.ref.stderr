error-bypass.cpp:6 (41-47): Cannot apply `&&` to arguments of type int and int[3]
  6 | int foo(int a, array<int, 3> b) { return a && b; }
    |                                          ^^^^^^
note: They should be both a primitive type (like an int or enum) or a register type.

error-bypass.cpp:7 (41-47): Cannot apply `&&` to arguments of type int and int[3]
  7 | int bar(int a, array<int, 3> b) { return a && b; }
    |                                          ^^^^^^
note: They should be both a primitive type (like an int or enum) or a register type.
